### DISCUSSION TITLE
Adding Support for other LIDAR

### DISCUSSION THREAD

**Discussion by krishna4104** [hippo5329/linorobot2_hardware#8] [Ideas]
Hello Sir I was able to implement the set repo without the LIDAR. 
Can u help me out fro implementing RPLIDAR A1 M8 as done by [here](https://github.com/kaiaai/firmware)

**Comment by hippo5329** [comment]
Sorry. I have seen kaia.ai and looked at their code before. They blend the communication between ROS2 and the eps32 robot into a combined message type of their own. It is possible to add more Lidar support by adapting their drivers. However, I don't have those Lidar like rplidar A1 to test. I have LD19 only. They are stable on my robots. I am busy with my own robots. I have no time and no interest to do these Lidar drivers porting.

Regards,
Thomas
